person.rb

```diff
@@ -0,0 +1,46 @@
+class Person
+  attr_writer :id
+  attr_accessor :name, :age
+
+  def initialize(age, _name: 'Unknown', _parent_permission: true)
+    @id = Random.rand(10...100)
+    @name = name
+    @age = age
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permission == true
+  end
```

Comment:
- [ ] <b>REQUIRED:</b> You have created a function `of_age?` to be used in cases like this, could you please try to use it. It would be something like the following but you can still always decide to use it whoever you want.


```ruby
def can_use_services? 
    of_age?() || @parent_permission
end
```

Reply:
Hi @rukundoeric,
Please can you be more specific about this request?